src/Microsoft.VisualBasic/tests/VB/Interaction.Windows.vb

```diff
@@ -0,0 +1,132 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System.Runtime.InteropServices
+Imports System.Security
+Imports System.Runtime.CompilerServices
+
+Namespace Microsoft.VisualBasic
+
+    Partial Public Module Interaction
```

Comment:
Please remove Interaction.Windows.vb if not referenced in this PR.

Reply:
It is needed for testing FileIO though I could simplify it. That is why I moved it to tests.

Reply:
FileIO tests should not depend on MsgBox.

Reply:
They depend on being about to commutate with the manual tester. MsgBox seemed line the easiest way to do it and when it becomes available enhance and simplify the tests.. I need to give instructions on what options to select and what actions to take for UI testing.

Reply:
I don't think we should include manual tests in these test classes.

Reply:
I don't understand the comment, I followed all the examples of manual testing from other CoreFx ports except they are using the console and I am using the GUI which is why I need MsgBox or equivalent. What are you suggesting, I could just create a local function based on MessageBox with a unique name.